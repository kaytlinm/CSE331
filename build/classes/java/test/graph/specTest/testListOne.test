#List One Test File
#Tests that a graph can be created, one node added and printed out.

CreateGraph graph1
AddNode graph1 n1
ListNodes graph1