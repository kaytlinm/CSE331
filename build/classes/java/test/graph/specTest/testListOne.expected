#List One Test File
#Tests that a graph can be created, one node added and printed out.

created graph graph1
added node n1 to graph1
graph1 contains: n1